style.css

```diff
@@ -3,7 +3,7 @@ body {
   overflow: scroll;
 }
 
-.calculator {
+#calculator {
```

Comment:
This works fine for a single calculator, but I think that when you get to stage 3, you'll probably want to change these back to classes instead of ID's.